app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,9 @@
+module CommentsHelper
+  def user_allowed_to_manage(comment)
```

Comment:
in ruby we have a convention - all methods that return boolean value should have `?` in their name `user_can_edit_comment?`